frontend/src/components/@common/Marquee/Marquee.tsx

```diff
@@ -0,0 +1,75 @@
+import { PropsWithChildren } from 'react';
+import styled from 'styled-components';
+
+interface MarqueeProps {
+  width: number;
+}
+
+function Marquee({ width, children }: PropsWithChildren<MarqueeProps>) {
+  return <StyledMarquee width={width}>{children}</StyledMarquee>;
+}
```

Comment:
PropsWithChildren 을 줬다는 것은 children 으로 받을 값을 제한하겠다는 의도가 맞을까요?
아니라면 Marquee 는 확장성 면에서 children 타입을 좀 더 크게 가져가는 것이 좋아보입니다.

```suggestion
interface MarqueeProps {
  width: number;
  children: React.ReactNode;
}

function Marquee({ width, children }: MarqueeProps) {
  return <StyledMarquee width={width}>{children}</StyledMarquee>;
}
```

Reply:
`PropsWithChildren`를 사용할 경우 children이 옵셔널로 된다는 것을 처음 알았네요.